jekyll/photos.html

```diff
@@ -0,0 +1,95 @@
+---
+layout: page
+---
+
+  <style>
+.lightbox-gallery{
+  background-repeat: no-repeat;
+  color: #000;
+  overflow-x: hidden
+}
+.lightbox-gallery p{color:#fff}.lightbox-gallery h2{font-weight:bold;margin-bottom:40px;padding-top:40px;color:#fff}@media (max-width:767px){.lightbox-gallery h2{margin-bottom:25px;padding-top:25px;font-size:24px}}.lightbox-gallery .intro{font-size:16px;max-width:500px;margin:0 auto 40px}.lightbox-gallery .intro p{margin-bottom:0}.lightbox-gallery .photos{padding-bottom:20px}.lightbox-gallery .item{padding-bottom:30px}
+  </style>
+
+ 
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.8.2/css/lightbox.min.css">
+
+<div class="alt-background">
+      <div class="container">
+        <div class="row title">
+          <div class="col-md-12">
+             <h1 class="rounded">Photos</h1>
+          </div>
+        </div>
+
+        <div class="lightbox-gallery">
+          <div class="container">
+           <div class="row photos">
+
+             <div class="col-sm-6 col-md-4 col-lg-3 item"><a href="2024/photos/full/IMG_9022.JPG" data-lightbox="photos"><img class="img-fluid" src="2024/photos/thumbnails/IMG_9022.JPG"></a></div>
```

Comment:
Given that all the images for this particular edition were under `img/`, why choose `2024/photos/` rather than `img/photos`? I'd love to be able to just move everything to the `2024/` subdir for archiving without breaking any links, and it seems as though that'd require a `2024/2024/photos/` subdir with the current file layout.

Reply:
That makes sense. I wanted to clearly show these were the 2024 foto's, but considering the archiving I'll strip off the 2024.